### PR TITLE
[APB-5103][LC]-wip

### PR DESCRIPTION
**Pull request by Leeroyc720**

### CODE REVIEW

File: app/uk/gov/hmrc/agentsexternalstubs/controllers/DesStubController.scala [hunk 1]
```diff
@@ -292,6 +292,10 @@ class DesStubController @Inject()(
               businessPartnerRecordsService
                 .getBusinessPartnerRecord(Utr(utr), session.planetId)
                 .flatMap(getOrCreateBusinessPartnerRecord(payload, idType, idNumber, session.planetId)(record => true))
+            case ("urn", urn) =>
```
Comment (geoffreywatson) [inline app/uk/gov/hmrc/agentsexternalstubs/controllers/DesStubController.scala:295]: do we need this? doesn't it belong to POST          /registration/individual/:idType/:idNumber   ?

File: app/uk/gov/hmrc/agentsexternalstubs/models/LegacyRelationshipRecord.scala [hunk 2]
```diff
@@ -32,7 +32,7 @@ case class LegacyRelationshipRecord(
       nino.map(LegacyRelationshipRecord.ninoKey),
       utr.map(LegacyRelationshipRecord.utrKey),
       Option(LegacyRelationshipRecord.agentIdKey(agentId)),
-      utr.map(LegacyRelationshipRecord.agentIdAndUtrKey(agentId, _))
+      utr.map(LegacyRelationshipRecord.agentIdAndUtrKey(agentId, _)),
```
Comment (geoffreywatson) [inline app/uk/gov/hmrc/agentsexternalstubs/models/LegacyRelationshipRecord.scala:35]: do we need this? legacy?

File: app/uk/gov/hmrc/agentsexternalstubs/controllers/DesStubController.scala (hunk 1 above)
Comment (Leeroyc720) [reply app/uk/gov/hmrc/agentsexternalstubs/controllers/DesStubController.scala:295]: you maybe right, I will have a look

File: app/uk/gov/hmrc/agentsexternalstubs/models/LegacyRelationshipRecord.scala (hunk 2 above)
Comment (Leeroyc720) [reply app/uk/gov/hmrc/agentsexternalstubs/models/LegacyRelationshipRecord.scala]: this was already there before I checked with master

**Pull request merged**